### PR TITLE
 Change to grep/sed solution due Mac grep difference

### PR DESCRIPTION
**Pull request by heliocastro**
Mac BSD based grep tool has different options and behaviour than Linux GNU based, so a solution with basic sed and grep was needed.

### CODE REVIEW

File: scripts/docker_build.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@
 DOCKER_ARGS=$@
 
 GIT_ROOT=$(git rev-parse --show-toplevel)
-GIT_REVISION=$($GIT_ROOT/gradlew -q properties --property version | grep -oP "version: \K.+")
+GIT_REVISION=$("$GIT_ROOT/gradlew" -q properties --property version | grep "version" | sed -e "s/version: //")
```
Comment (sschuberth) [inline scripts/docker_build.sh:23]: Some things, please
- move the addition of quotes to a separate commit, which a separate rationale,
- explain what part of the `grep`command wasn't working on macOS,
- use either `grep` or `sed`, but not both.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: scripts/docker_build.sh (hunk 1 above)
Comment (heliocastro) [reply scripts/docker_build.sh]: Fixed

File: scripts/docker_build.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@
 DOCKER_ARGS=$@
 
 GIT_ROOT=$(git rev-parse --show-toplevel)
-GIT_REVISION=$($GIT_ROOT/gradlew -q properties --property version | grep -oP "version: \K.+")
+GIT_REVISION=$($GIT_ROOT/gradlew -q properties --property version |  awk '/version: / {print $2}')
```
Comment (sschuberth) [inline scripts/docker_build.sh:23]: Please no heavy-weight `awk` either. How about `sed -nr "s/version: (.+)/\1/p"`, does that work on macOS?

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: scripts/docker_build.sh (hunk 1 above)
Comment (sschuberth) [reply scripts/docker_build.sh]: There's now a merge commit in the PR.

File: scripts/docker_build.sh (hunk 2 above)
Comment (heliocastro) [reply scripts/docker_build.sh:23]: Works, but i need just disagree about "heavy-weight" awk, when you need entire gradlew and parsing the entire project in the same line before :-)

Comment (sschuberth) [reply scripts/docker_build.sh]: Awk is a new dependency, Gradle is not. If it's in use already anyway, it doesn't matter that much how heavy-weight the tool is. And Gradle we bootstrap, whereas Awk might not be available on all platforms, like Windows (via WSL or Git Bash).

File: scripts/docker_build.sh [hunk 3]
```diff
@@ -20,7 +20,7 @@
 DOCKER_ARGS=$@
 
 GIT_ROOT=$(git rev-parse --show-toplevel)
-GIT_REVISION=$($GIT_ROOT/gradlew -q properties --property version | grep -oP "version: \K.+")
+GIT_REVISION=$($GIT_ROOT/gradlew -q properties --property version |  sed -nr "s/version: (.+)/\1/p")
```
Comment (sschuberth) [inline scripts/docker_build.sh:23]: Double-space before `sed`.

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Review by sschuberth — APPROVED** [review]

File: scripts/docker_build.sh (hunk 3 above)
Comment (heliocastro) [reply scripts/docker_build.sh]: Done
